zealot/util/utils.ts

```diff
@@ -54,3 +55,23 @@ export function uniq(array: any[]) {
   }
   return u
 }
+
+export const nodeStreamToWebStream = (
```

Comment:
Not using this yet, but will need it in the brimcap plugin when we pipe the `brimcap analyze` output to the backend's /add endpoint